packages/slice-machine/server/src/index.ts

```diff
@@ -95,6 +96,21 @@ app.use("/onboarding", (_, res) => {
 });
 
 const PORT = process.env.PORT || "9999";
+
+app.use(
+  (
+    err: Error,
+    _req: express.Request,
+    res: express.Response,
+    // This is need becuase express middleware uses arguments.length
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _next: express.NextFunction
+  ) => {
+    console.error(err);
```

Comment:
Just asking because I don't know but will that display the error twice ?

Reply:
It's there to handle unhandled errors/throws,
but if an error is logged and thrown again, then yes.

Reply:
Ok i see, then yes i guess if it catches an unhandled error then it would not have been logged before, good.